acme.sh

```diff
@@ -4200,7 +4200,24 @@ __purge_txt() {
 _check_dns_entries() {
   _success_txt=","
   _end_time="$(_time)"
-  _end_time="$(_math "$_end_time" + 1200)" #let's check no more than 20 minutes.
+
+  # Use default values for DNS timeout (1200 seconds/20 minutes) and DNS retry
+  # interval (10 seconds), unless configured at script invocation to be
+  # something else.
+  _dnstimeout=1200 #default timeout is 20 minutes
+  if [ -n "$Le_DNSValidateTimeout" ]; then
```

Comment:
move this `_savedomainconf` to the `_issue()` function.
The `save` operation should only happen in `_issue()` method